test/Maestro.ScenarioTests/MaestroScenarioTestBase.cs

```diff
@@ -216,33 +216,33 @@ internal async Task<AsyncDisposableValue<Microsoft.DotNet.DarcLib.PullRequest>>
             throw new MaestroTestException($"The created pull request for {targetRepoName} targeting {targetBranch} was not updated with subsequent subscriptions after creation");
         }
 
-        public async Task CheckBatchedGitHubPullRequest(string targetBranch, string source1RepoName, string source2RepoName,
+        public async Task CheckBatchedGitHubPullRequest(string targetBranch, string[] sourceRepoNames,
             string targetRepoName, List<Microsoft.DotNet.DarcLib.DependencyDetail> expectedDependencies, string repoDirectory)
         {
-            string[] expectedPRTitles = [
-                $"[{targetBranch}] Update dependencies from {_parameters.GitHubTestOrg}/{source1RepoName} {_parameters.GitHubTestOrg}/{source2RepoName}",
-                $"[{targetBranch}] Update dependencies from {_parameters.GitHubTestOrg}/{source2RepoName} {_parameters.GitHubTestOrg}/{source1RepoName}",
-            ];
+            var repoNames = sourceRepoNames
+                .Select(name => $"{_parameters.GitHubTestOrg}/{name}")
+                .OrderBy(s => s);
```

Comment:
nit: maybe `r` is better than `s` here, but I don't really mind

Reply:
.NET 8.0 and `Order()` would be the best :)